examples/cifar10/README.md

```diff
@@ -1,260 +1,11 @@
-| cifar10_central | 1.0	| 	0.8798	| 
-| cifar10_fedavg  | 1.0	| 	0.8854	| 
-| cifar10_fedavg_he | 1.0	| 	0.8897	|
-
-![Central vs. FedAvg](./figs/central_vs_fedavg_he.png)
-
-### 4.2 Impact of client data heterogeneity
-
-We also tried different `alpha` values, where lower values cause higher heterogeneity. 
-This can be observed in the resulting performance of the FedAvg algorithms.  
-
-| Config |	Alpha |	Val score |
-| ----------- | ----------- |  ----------- |
-| cifar10_fedavg |	1.0 |	0.8854 |
-| cifar10_fedavg |	0.5 |	0.8633 |
-| cifar10_fedavg |	0.3 |	0.8350 |
-| cifar10_fedavg |	0.1 |	0.7733 |
-
-![Impact of client data heterogeneity](./figs/fedavg_alpha.png)
-
-### 4.3 FedAvg vs. FedProx vs. FedOpt vs. SCAFFOLD
-
-Finally, we compare an `alpha` setting of 0.1, causing a high client data heterogeneity and its 
-impact on more advanced FL algorithms, namely FedProx, FedOpt, and SCAFFOLD. FedProx and SCAFFOLD achieve better performance compared to FedAvg and FedProx with the same `alpha` setting. However, FedOpt and SCAFFOLD show markedly better convergence rates. SCAFFOLD achieves that by adding a correction term when updating the client models, while FedOpt utilizes SGD with momentum
-to update the global model on the server. Both achieve better performance with the same number of training steps as FedAvg/FedProx.
-
-| Config           |	Alpha |	Val score |
-|------------------| ----------- |  ---------- |
-| cifar10_fedavg   |	0.1 |	0.7733 |
-| cifar10_fedprox  |	0.1 |	0.7615 |
-| cifar10_fedopt   |	0.1 |	0.8013 |
-| cifar10_scaffold |	0.1 |	0.8222 |
-
-![FedProx vs. FedOpt](./figs/fedopt_fedprox_scaffold.png)
-
-
-## 5. Streaming TensorBoard metrics to the server
-
-In a real-world scenario, the researcher won't have access to the TensorBoard events of the individual clients. In order to visualize the training performance in a central place, `AnalyticsSender`, `ConvertToFedEvent` on the client, and `TBAnalyticsReceiver` on the server can be used. For an example using FedAvg and metric streaming during training, run:
-```
-./submit_job.sh cifar10_fedavg_stream_tb 1.0
-```
-Using this configuration, a `tb_events` folder will be created under the `[JOB_ID]` folder of the server that includes all the TensorBoard event values of the different clients.
+### [Simulated Federated Learning with CIFAR-10](./cifar10-sim/README.md)
+This example includes instructions on running [FedAvg](https://arxiv.org/abs/1602.05629), 
+[FedProx](https://arxiv.org/abs/1812.06127), [FedOpt](https://arxiv.org/abs/2003.00295), 
+and [SCAFFOLD](https://arxiv.org/abs/1910.06378) algorithms using NVFlare's FL simulator.
+
+### [Real-world Federated Learning with CIFAR-10](./cifar10-real-world/README.md)
+Includes instructions on running [FedAvg](https://arxiv.org/abs/1602.05629) with streaming 
+of TensorBoard metrics to the server during training 
+and [homomorphic encryption](https://developer.nvidia.com/blog/federated-learning-with-homomorphic-encryption/).
```

Comment:
maybe some more explanation about "real-world" ("as compared with simulator, real-world application requires provisioning etc. ....")?

Reply:
or just move the "For more information on real-world FL see [here](https://nvflare.readthedocs.io/en/latest/real_world_fl.html)." sentence here

Reply:
Good point. I put some explaination here as well.